### PR TITLE
Add 2.0 proposed OS version support

### PR DESCRIPTION
**Pull request by Petermarcu**
@hqueue, can you comment on the minimum Tizen version?

### CODE REVIEW

**Comment by Petermarcu** [comment]
@leecow can you review?
@hqueue can you help on the minimum Tizen version?

File: roadmap.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Architecture:
 - [.NET Standard](https://github.com/dotnet/standard)
 - [.NET Core Tools/CLI](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/index)
 
-Version 2.0 minumum OS Support (Proposed):
+Version 2.0 minimum OS Support (Proposed):
 
 OS                            |Version                        |Architectures|Configurations|Notes
```
Comment (leecow) [inline roadmap.md]: Ubuntu 17.04 is expected in April. Possibly add a note that support is pending official release? Same would apply to macOS 10.13 and Debian 9.

Comment (Petermarcu) [inline roadmap.md]: I was trying to intentionally just list the minimum for any given family. Do you think it should be version ranges? Fedora 26 would also be in that list.

**Comment by hqueue** [comment]
/cc: @lemma @jyoungyun Please take a look at Tizen version and architecture.

File: roadmap.md [hunk 2]
```diff
@@ -99,6 +99,20 @@ Architecture:
...
+Version 2.0 minimum OS Support (Proposed):
+
+OS                            |Version                        |Architectures|Configurations|Notes
+------------------------------|-------------------------------|-------------|--------------|-----
+Windows Client                | 7 SP1 - 10                    | x64, x86    |              |
+Windows Server                | 2008 R2 SP1 - 2016            | x64, x86    | Full, Server Core, Nano |
+Red Hat Enterprise Linux      | 7.2                           | x64         |              | This includes Centos and Oracle Linux
+Fedora                        | 25                            | x64         |              |
+Debian                        | 8.2                           | x64         |              |
+Ubuntu                        | 14.04, 16.04                  | x64         |              | This includes Linux Mint 17, 18
```
Comment (jyoungyun) [inline roadmap.md]: How about Ubuntu arm? Could you please let me know the reason why you didn't add arm architecture here?

File: roadmap.md [hunk 3]
```diff
@@ -99,6 +99,20 @@ Architecture:
...
+openSUSE                      | 42.2                          | x64         |              |
+Tizen                         | TBD                           | x64, arm32  |              |
```
Comment (jyoungyun) [inline roadmap.md]: And.. We do not support Tizen x64  yet.

**Comment by lemmaa** [comment]
@Petermarcu Tizen's first official .NET support is from version 4.0, which will be released this fall. This will supported from NuGet 4.3.0 client. As for the target architecture, currently Tizen only supports arm32. x64(or x86) will only be supported as an emulator environment for development convenience in the SDK.

**Comment by WonyoungChoi** [comment]
In addition, The TFM "Tizen 4" can be used from NuGet 4.3.0. https://github.com/NuGet/NuGet.Client/pull/1232

**Comment by jyoungyun** [comment]
Here is the patches for Tizen nupkgs. These PR supported to generate Tizen.4.0.0 nupkgs.
https://github.com/dotnet/coreclr/pull/8827
https://github.com/dotnet/corefx/pull/14792
https://github.com/dotnet/core-setup/pull/1025

**Comment by hqueue** [comment]
@Petermarcu In short, 4.0 is minimum version. And please remove x64 from Tizen architecture.

**Comment by Petermarcu** [comment]
Thanks all for the feedback. I have updated the PR to take it all into account.

**Pull request merged**